src/index.html

```diff
@@ -8,6 +8,84 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars-container">
+      <div class="stars stars--0">
+        <div class="stars__empty">
+        </div>
+        <div class="stars__empty">
+        </div>
+        <div class="stars__empty">
+        </div>
+        <div class="stars__empty">
+        </div>
+        <div class="stars__empty">
+        </div>
```

Comment:
<img width="607" alt="image" src="https://github.com/mate-academy/layout_stars/assets/102411130/9abadc12-138d-456b-8b65-1a88fc247276">
you should implement count of active stars with css styles